mlflow/pyfunc/scoring_server/client.py

```diff
@@ -2,6 +2,19 @@
 import time
 from mlflow.pyfunc import scoring_server
 import json
+import datetime
+
+# Extended `DateTimeEncoder` in https://stackoverflow.com/a/12126976 to support `pandas.TimeStamp`
```

Comment:
# Extended `DateTimeEncoder` in https://stackoverflow.com/a/12126976 to support `pandas.TimeStamp`
```suggestion
# Extended `json.JSONEncoder` to support `pandas.TimeStamp`, reference: https://stackoverflow.com/a/12126976
```

Reply:
Fixed.